app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -278,7 +293,7 @@ export default {
       if (image) {
         return `/file/${image}`
       }
-      return `/images/pages/league/graphic_1.png`
+      return `/images/pages/league/student_hugging.png`
```

Comment:
We should either rename the images to have the same names as before, or we should remove the old unused images (I think game_hero.png, graphic_1.png, and graphic_hugging.png).

Reply:
Removed unused images `game_hero.png`, `graphic_1.png`, and `graphic_hugging.png`